packaging/debian/changelog.in

```diff
@@ -1 +1,5 @@
 cockpit-navigator (VERSION-1) UNRELEASED; urgency=medium
+
+  * Upstream build
+
+ -- Cockpit <cockpit-devel@lists.fedorahosted.org>  Wed, 27 Sep 2023 13:34:21 +0200
```

Comment:
This belongs into the first commit, not "Add Makefile rule to generate debian changelog". I.e. rebase error.

Reply:
Uh hmm, I don't think that can be a rebase issue but shall I just squash it to a generic add packaging commit.